packages/suite/src/components/wallet/CoinjoinSummary/CoinjoinStatus.tsx

```diff
@@ -158,6 +159,29 @@ export const CoinjoinStatus = ({
             {
                 key: 'coinjoin-actions',
                 options: [
+                    {
+                        key: 'resume',
+                        label: (
+                            <MenuLabel>
+                                <Icon icon="PLAY" size={10} />
+                                <Translation id="TR_RESUME" />
+                            </MenuLabel>
+                        ),
+                        callback: togglePause,
+                        'data-test': `@coinjoin/resume`,
+                        isHidden: !isPaused || isLoading,
+                    },
+                    {
+                        key: 'resuming',
```

Comment:
Clicking the "resuming" item closes the dropdown. I would expect a default cursor, grey text and no action on click. But no big deal :)
![chrome-capture-2022-9-31](https://user-images.githubusercontent.com/42465546/198970597-6b9625a1-c3e2-4748-9976-eed3e4162239.gif)

Reply:
3edbcd55e